molecule/kafka-connect-replicator-mtls-scram-rhel/converge.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Remove the Zookeeper, Kafka Broker Groups from Inventory
-  hosts: zookeeper:kafka_broker:schema_registry:kafka_rest
+  hosts: zookeeper:kafka_connect:kafka_broker:schema_registry:kafka_rest
```

Comment:
Is this intended - adding kafka_connect here?

Reply:
It should be kafka_controller. Correcting this.